.github/workflows/ci.yml

```diff
@@ -149,18 +149,15 @@ jobs:
           RUST_BACKTRACE: 1
         run: nix develop -L --no-update-lock-file --command just impure-tests
 
+      # This differs from `nix-build-bats-tests` in that:
+      #   - a cargo-built debug binary is used, like you would in local development
+      #   - envrionment variables unskip some tests against live systems
       - name: "CLI Integration Tests"
-        env:
-          RUST_BACKTRACE: 1
-        run: nix develop -L --no-update-lock-file --command just integ-tests
-
-      - name: "Functional Tests"
-        if: ${{ github.event_name == 'merge_group' || github.event_name == 'workflow_dispatch' }}
         env:
           RUST_BACKTRACE: 1
           AUTH0_FLOX_DEV_CLIENT_SECRET: "${{ secrets.MANAGED_AUTH0_FLOX_DEV_CLIENT_SECRET }}"
-          TESTING_FLOX_CATALOG_URL: "${{ secrets.MANAGED_TESTING_FLOX_CATALOG_URL }}"
-        run: nix develop -L --no-update-lock-file --command just functional-tests
+          TESTING_FLOX_CATALOG_URL: "https://flox-catalog.flox.dev"
```

Comment:
Suggestion: I don't think we should switch to production here. When we make API changes those are first deployed to the preview instance so that we can test against the changes. If we test against production then CI won't work with the new client, etc until the preview changes are deployed to production. I'd rather we have a green light from CI before deploying anything to production.

Reply:
Fair point. I'll revert the commit now that preview is fixed.

We're still going to need something else (e.g. some other test strategy) to determine whether promoting catalog changes from preview to production are going to be safe but that's beyond scope for now.